storage/country_info_getter.hpp

```diff
@@ -88,6 +88,9 @@ class CountryInfoGetter
   // Returns true when |pt| belongs to a country identified by |id|.
   virtual bool IsBelongToRegionImpl(size_t id, m2::PointD const & pt) const = 0;
 
+  // Return true when distance from |pt| to country identified by |id| less then |distance|
```

Comment:
// Returns true when the distance from |pt| to country identified by |id| less then |distance|.